user_install.sh

```diff
@@ -50,6 +50,7 @@ check_host() {
 #=======================#
 
 CATKIN_DIR=~/mil_ws
+MIL_CONFIG_DIR=~/.mil
 BASHRC_FILE=~/.bashrc
 MILRC_FILE=~/.milrc
```

Comment:
As another point, this would cause problems for anyone that has run the script previously. If I have the install script create the `~/.mil/milrc` file and append the line to source it to the user's `~/.bashrc` file, they will still be sourcing the old file as well. As things change in the new file, this could come back to haunt us.